lib/matplotlib/text.py

```diff
@@ -825,10 +816,10 @@ def set_backgroundcolor(self, color):
 
         ACCEPTS: any matplotlib color
         """
-        if self._bbox is None:
-            self._bbox = dict(facecolor=color, edgecolor=color)
+        if self._bbox_patch is None:
+            self.set_bbox(dict(facecolor=color, edgecolor=color))
         else:
-            self._bbox.update(dict(facecolor=color))
+            self._bbox_patch.update(dict(facecolor=color))
```

Comment:
I don't remember exactly what I was thinking when I left the last message.  I think what I was suggesting was to get rid of the `else` block and just always update the `_bbox_patch` (even if we just created it with the correct color).  It is a no-op and I think makes the code path a little easier to follow (which is a style call).

Avoiding `else` statements and having `if` blocks which just ensure assumptions are true is is a newish (and possibly not good) style-tick I have developed. 

I think my comment on this can be ignored.